app/assets/stylesheets/_sn.scss

```diff
@@ -799,3 +799,9 @@
     }
   }
 }
+
+.dimmed {
```

Comment:
There are https://github.com/standardnotes/web/blob/develop/app/assets/stylesheets/_ui.scss#L80 (`faded`: opacity 0.5) and https://github.com/standardnotes/web/blob/develop/app/assets/stylesheets/_ui.scss#L205 (`cursor-default`) classes that you can use instead of this I think. And then maybe create a `pointer-events-none` utility class, so that it's more in line with the use of utility classes in the webapp. Wdyt?